hook.php

```diff
@@ -475,3 +483,24 @@ function plugin_formcreator_dynamicReport($params) {
 
    return false;
 }
+
+/**
+ * Hook for timeline_actions; display a new action for a CommonITILObject
+ * @see CommonITILObject
+ *
+ * @return void
+ */
+function plugin_formcreator_timelineActions($options) {
+   $item = $options['item'];
+   if (!$item->canDeleteItem()) {
```

Comment:
@orthagh : It looks like this method is a better choice than canRequesterUpdateItem() (by reading Ticket::showForm() )

Reply:
seems legit. I think there is in this function a repetition of code with canRequesterUpdateItem. It may need a factorization between the two on the core.